source/help/mirrors.md

```diff
@@ -1,14 +1,6 @@
 arXiv mirror sites
 ==================
 
-In addition to the [main site](https://arxiv.org), there are mirror sites for arXiv content. These are
-updated daily but have fewer features than the main site:
+The mirror network has been shut down as of September 15th, 2024.
 
--   [cn.arXiv.org](http://cn.arxiv.org/) (China)
--   [de.arXiv.org](http://de.arxiv.org/) (Germany)
--   [in.arXiv.org](http://in.arxiv.org/) (India)
--   [lanl.arXiv.org](http://lanl.arxiv.org/) (née xxx.lanl.gov, U.S. mirror at Los Alamos)
-
-
-We are in the process of discontinuing the arXiv mirror network. We will continue to support and extend [APIs and data feeds](bulk_data.md) to enable third
-parties to develop services over arXiv.
+The arXiv mirrors were begun as an attempt to provide better local access to the arXiv corpus to facilitate speed during the nascent modern internet. The mirror network was significantly expanded in the late 1990's and, at the time, was heavily used. Geographic locality is now much less useful than it was when the mirror network was established. Now, most users geographically close to mirrors, perhaps in neighboring countries, have better access to the main site than to a "nearby" mirror. These read-only mirrors took significant effort of both arXiv staff and hosting volunteer institutions to maintain, implement and update. Over time, their utility and utilization has dwindled to near zero. arXiv’s current technology plan has us moving to a fully cloud-based system in the coming months, rendering the intended purpose of the mirrors no longer relevant. 
```

Comment:
suggest removing
"Over time, their utility and utilization has dwindled to near zero."
That is already covered by the following sentence.